File: petManage_api/src/main/java/dot/petstore/model/Funcionario.java
package dot.petstore.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Funcionario {
    private int idFuncionario;
    private String nome;
    private String endereco;
    private String senha;
    private String cpf;
    private String email;
    private String telefone;
    private boolean gerente;
}

File: petManage_app/src/app/model/funcionario.ts
export class Funcionario {
    idFuncionario: number;
    nome: string;
    cpf: string;
    telefone: string;
    gerente: boolean;
    endereco: string;
    email: string;
    senha: string;

    constructor(){
        this.idFuncionario = 0;
        this.nome  = "";
        this.cpf = "";
        this.telefone = "";
        this.gerente = false;
        this.endereco = "";
        this.email = "";
        this.senha = "";
    }
}